locales/settings/zh-TW.json

```
@@ -0,0 +1,195 @@
+  "invitation.earlyAccess": "Follow 目前處於 <strong>早期開發</strong>狀態，需要邀請碼才能使用。",
+  "invitation.earlyAccessMessage": "😰 抱歉，關注目前處於搶先體驗階段，需要邀請碼才能使用。",
+  "invitation.generateButton": "產生邀請碼",
+  "invitation.generateCost": "您可以花費 {{INVITATION_PRICE}} <PowerIcon>Power</PowerIcon> 為您的朋友產生邀請碼。",
+  "invitation.getCodeMessage": "您可以通過以下方式獲取邀請碼：",
+  "invitation.newInvitationSuccess": "🎉 邀請碼已產生, 邀請碼已複製",
+  "invitation.noInvitations": "沒有邀請",
+  "invitation.notUsed": "為使用",
+  "invitation.sidebar_title": "邀請",
+  "invitation.tableHeaders.code": "代碼",
+  "invitation.tableHeaders.creationTime": "建立時間",
+  "invitation.tableHeaders.usedBy": "使用者",
+  "invitation.title": "邀請碼",
+  "profile.avatar.label": "大頭貼",
+  "profile.handle.description": "您的唯一識別碼",
+  "profile.handle.label": "Handle",
+  "profile.name.description": "您的公開顯示名稱。",
+  "profile.name.label": "顯示名稱",
+  "profile.sidebar_title": "個人資料",
+  "profile.submit": "送出",
+  "profile.title": "配置文件設定",
+  "profile.updateSuccess": "個人資料已更新。",
+  "titles.about": "關於",
+  "titles.actions": "指令",
+  "titles.appearance": "外觀",
+  "titles.general": "一般",
+  "titles.integration": "整合",
+  "titles.invitations": "邀請",
+  "titles.power": "Power",
+  "titles.profile": "個人資料",
+  "titles.shortcuts": "快捷鍵",
+  "wallet.address.title": "您的地址",
+  "wallet.balance.title": "您的餘額",
+  "wallet.balance.withdrawable": "可提取",
+  "wallet.balance.withdrawableTooltip": "可提取的 Power 包括你收到的贊助和充值的 Power。",
+  "wallet.claim.button.claim": "領取每日 Power",
+  "wallet.claim.button.claimed": "今日已領取",
+  "wallet.claim.tooltip.alreadyClaimed": "您今天已經領取過了。",
+  "wallet.claim.tooltip.canClaim": "立即領取您的 {{amount}} 每日 Power！",
+  "wallet.create.button": "建立錢包",
+  "wallet.create.description": "建立一個免費錢包以接收 <PowerIcon /> <strong>Power</strong>，可用於獎勵創作者，也可以因您的內容貢獻而獲得獎勵。",
+  "wallet.power.dailyClaim": "您每天可以領取 {{amount}} 個免費 Power，可用於在 Follow 上贊助 RSS 項目。",
+  "wallet.power.description": "Power 是 {{blockchainName}} 區塊鏈上的一種 <Link>ERC-20 代幣</Link>。",
+  "wallet.sidebar_title": "Power",
+  "wallet.transactions.amount": "數量",
+  "wallet.transactions.date": "日期",
+  "wallet.transactions.from": "發送至",
+  "wallet.transactions.noTransactions": "無交易紀錄",
+  "wallet.transactions.title": "交易紀錄",
+  "wallet.transactions.to": "接收自",
+  "wallet.transactions.tx": "交易",
+  "wallet.transactions.type": "類型",
+  "wallet.transactions.types.burn": "burn",
+  "wallet.transactions.types.mint": "mint",
+  "wallet.transactions.types.purchase": "purchase",
+  "wallet.transactions.types.tip": "tip",
+  "wallet.transactions.types.withdraw": "withdraw",
+  "wallet.transactions.you": "您",
+  "wallet.withdraw.addressLabel": "您的以太坊地址",
+  "wallet.withdraw.amountLabel": "數量",
+  "wallet.withdraw.availableBalance": "您的錢包中有 <Balance></Balance> 可提取 Power。",
+  "wallet.withdraw.button": "提取",
+  "wallet.withdraw.error": "提取失败：{{error}}",
+  "wallet.withdraw.modalTitle": "提取 Power",
+  "wallet.withdraw.submitButton": "送出",
+  "wallet.withdraw.success": "送出成功！"
```

```suggestion
  "wallet.withdraw.success": "提取成功！"
```
https://github.com/RSSNext/Follow/blob/e1233a3f008abaf0dc4102d9d20777e77f27f1c7/locales/settings/en.json#L194